subprojects/antlr/src/main/java/org/gradle/api/plugins/antlr/AntlrPlugin.java

```diff
@@ -54,10 +55,8 @@ public void apply(final Project project) {
 
         // set up a configuration named 'antlr' for the user to specify the antlr libs to use in case
         // they want a specific version etc.
-        final Configuration antlrConfiguration = project.getConfigurations().create(ANTLR_CONFIGURATION_NAME)
+        final Configuration antlrConfiguration = ((ProjectInternal) project).getConfigurations().resolvableBucket(ANTLR_CONFIGURATION_NAME)
```

Comment:
It seems we are not using the overload which enables warnings on role changes. Is that going to be in a follow-up PR? I imagine (as I think you mentioned) that a lot of our own integration tests are affected here.

Reply:
Yes, there's enough here already, I'll do warning changes in a followup.